src/List.js

```diff
@@ -7,7 +7,12 @@ const List = (props) => {
             {groceries.map((item) => {
                 return (
                     <div key={item.id}>
-                        <ListItem key={item.id} value={item.value} handleDelete={()=>{props.handleDelete()}} />
+                        <ListItem
+                            key={item.id}
+                            id={item.id}
+                            value={item.value}
+                            handleDelete={props.handleDelete}
```

Comment:
Dit is hetzelfde als `handleDelete={(id)=>{props.handleDelete(id)}}`. Omdat props.handleDelete al een functie is hoef je er niet nog een aparte function omheen te zetten. Dus nu wordt props.handleDelete direct doorgegeven aan ListItem.